server/channels/app/post.go

```diff
@@ -480,6 +487,12 @@ func (a *App) handlePostEvents(c request.CTX, post *model.Post, user *model.User
 	if channel.TeamId != "" {
 		t, err := a.Srv().Store().Team().Get(channel.TeamId)
 		if err != nil {
+			a.NotificationsLog().Error("Missing team",
+				mlog.String("post_id", post.Id),
+				mlog.String("status", model.StatusServerError),
+				mlog.String("reason", model.ReasonFetchError),
+				mlog.Err(err),
+			)
```

Comment:
At this level, are we certain that this post would've generated a notification? It seems we may be adding much more logging errors that may not be tied to a notification and cause noise/confusion in the notifications log

Reply:
At this point we have created a post so if this call fails we would not generate a notification, but we should be. This is pretty unlikely to fail but it is important to call out if we're getting missing notifications. One case where it could fail is data sync issues between clusters, or if the admin has gone and messed with the data manually.